fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -298,9 +298,12 @@ public void postDruidQuery(
     ) {
         String entityBody;
         try {
+            RequestLog.startTiming("DruidQuerySerialization");
             entityBody = writer.writeValueAsString(druidQuery);
         } catch (JsonProcessingException e) {
             throw new IllegalStateException(e);
+        } finally {
+            RequestLog.stopTiming("DruidQuerySerialization");
```

Comment:
This should include the sequence number (see the timing we do below). Otherwise, when stitching requests back together, we may have trouble.